src/core/fem/src/condition/4C_fem_condition.hpp

```diff
@@ -32,6 +32,27 @@ namespace Core::Elements
 
 namespace Core::Conditions
 {
+  /**
+   * Which type of entity is the ID of a condition referring to?
+   */
+  enum class EntityType
+  {
+    /**
+     * Refers to a numbering of nodes specified in the input file directly. Note that an ID is only
+     * unique within a GeometryType. The same ID may be used across different GeometryTypes.
+     */
+    legacy_id,
+
+    /**
+     * Refers to a node set ID.
+     */
+    node_set_id,
+
+    /**
+     * Refers to an element block ID.
+     */
+    element_block_id,
```

Comment:
For the future: Exodus also offers "side sets". Although `pre_exodus` had some side-set-code in it, we never used them in 4C. This might be worthwhile to investigate in the future to see, if and how side sets are superior and simpler in specifying boundary conditions on surfaces.